styles/styles.css

```diff
@@ -0,0 +1,178 @@
+body {
+  margin: 0;
+}
+
+ul {
+  list-style-type: none;
+  padding: 0;
+}
+
+.nav {
+  display: grid;
+  grid-template-columns: 4fr 1fr 1fr;
+  grid-template-rows: 50px;
+  justify-items: stretch;
+  align-items: center;
+  font-size: 1.2em;
+  font-weight: bold;
+  margin-left:100px;
+}
+
+.tagline {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-content: center;
+  text-align: center;
+  padding: 200px;
+}
+
+h1 {
+  font-size: 3em;
+}
+
+sub {
+  line-height: normal;
+  position: static;
+  font-weight: bold;
+  font-size: 2em;
+  flex: 1/1/200px;
+  padding:30px;
+
+}
+
+.button {
+    color: black;
+    font-size: 1.5em;
+    cursor: pointer;
```

Comment:
I love this!! A subtle additional of adding the hand pointer makes such a difference from the user's perspective!